app/app.go

```diff
@@ -563,6 +545,16 @@ func NewRollapp(
 	transferStack = transfer.NewIBCModule(app.TransferKeeper)
 	transferStack = claims.NewIBCMiddleware(*app.ClaimsKeeper, transferStack)
 	transferStack = erc20.NewIBCMiddleware(app.Erc20Keeper, transferStack)
+	transferStack = hubgenkeeper.NewIBCModule(
+		transferStack,
+		func(ctx sdk.Context, transfer *ibctransfertypes.MsgTransfer) error {
```

Comment:
can we pls use the original signature instead of wrapping

Reply:
sure I will change